co-log/src/Colog/Message.hs

```diff
@@ -503,7 +503,6 @@ builderDmyHMSz (C.OffsetDatetime (C.Datetime date time) offset) =
 
 {- | Shows a thread id in the following format:
 
->>> showThreadId <$> Control.Concurrent.myThreadId
```

Comment:
You don't need to delete this line. The documentation still can be helpful 🙂 
Usual trick in such situations is to replace `>>>` with `>>` and optionally make it **bold** for prettier printing 💅🏻 

```haskell
__>>__ showThreadId <$> Control.Concurrent.myThreadId

Reply:
Makes sense; wasn't aware of this trick.